server/events.lua

```diff
@@ -8,15 +8,19 @@ AddEventHandler('chatMessage', function(_, _, message)
 end)
 
 AddEventHandler('playerDropped', function(reason)
-    local src = source
-    if not QBCore.Players[src] then return end
-    local Player = QBCore.Players[src]
-    TriggerEvent('qb-log:server:CreateLog', 'joinleave', 'Dropped', 'red', '**' .. GetPlayerName(src) .. '** (' .. Player.PlayerData.license .. ') left..' ..'\n **Reason:** ' .. reason)
-    Player.Functions.Save()
-    QBCore.Player_Buckets[Player.PlayerData.license] = nil
-    QBCore.Players[src] = nil
+    CreateThread(function()
```

Comment:
Change this to SetTimeout and remove the wait and add `10000` after the `end` that closes the function to specify the timeout